lib/modules/showImages.js

```diff
@@ -1788,7 +1798,7 @@ function videoAdvanced(options) {
 
 	setMediaMaxSize(vid);
 	makeMediaZoomable(vid);
-	setMediaControls(vid);
+	setMediaControls(vid, null, options.sources[0].source);
```

Comment:
Nit: in general we prefer `undefined` to `null` for omitted params so that default param values work (if there were any)